src/booleans.js

```diff
@@ -1,64 +1,92 @@
+
 function negate(a) {
-  // your code here
+  return !a;
 };
 
 function both(a, b) {
-  // your code here
+  return a && b;
 };
 
 function either(a, b) {
-  // your code here
+  return a || b; 
 };
 
 function none(a, b) {
-  // your code here
+  if (a === false && b === false) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function one(a, b) {
-  // your code here
+  if (a === true && b === false) {
+    return true;
+  } else if (a === false && b === true) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function truthiness(a) {
-  // your code here
-};
+  return Boolean(a);
+}; 
 
 function isEqual(a, b) {
-  // your code here
+  return a === b;
 };
 
 function isGreaterThan(a, b) {
-  // your code here
+  return a > b;
 };
 
 function isLessThanOrEqualTo(a, b) {
-  // your code here
+  return a <= b;
 };
 
 function isOdd(a) {
-  // your code here
+  if (a % 2 == 1 ) {
+    return true; 
+  } else {
+    return false;
+  }
 };
 
 function isEven(a) {
-  // your code here
+  if (a % 2 == 0 ) {
+    return true; 
+  } else {
+    return false;
+  }
 };
 
 function isSquare(a) {
-  // your code here
+  return Number.isInteger(Math.sqrt(a));
 };
 
 function startsWith(char, string) {
-  // your code here
+  if (string[0] === char) {
+    return true; 
+  } else {
+    return false;
+  }
 };
 
 function containsVowels(string) {
-  // your code here
+  let pattern = /[aeiou]/i;
+  return pattern.test(string);
```

Comment:
Could condense this a bit and save creating a variable:
```suggestion
  return /[aeiou]/i.test(string);
```